featuretools/tests/entityset_tests/test_es.py

```diff
@@ -1514,3 +1514,18 @@ def test_entityset_init():
     es_copy.add_relationship(relationship)
     assert es['cards'].__eq__(es_copy['cards'], deep=True)
     assert es['transactions'].__eq__(es_copy['transactions'], deep=True)
+
+
+def test_add_interesting_values_verbose_output(caplog):
+    es = ft.demo.load_retail(nrows=200)
+    es['order_products'].convert_variable_type('quantity', ft.variable_types.Discrete)
+    logger = logging.getLogger('featuretools')
+    logger.propagate = True
+    logger = logging.getLogger('featuretools.entityset')
```

Comment:
if we use a separate variable for the second logger we don't need to call getLogger again later

Reply:
Updated